### PR TITLE
Add key mappings and legend to import page

### LINKED ISSUE #74: Add tutorial mode
From Brian's 17 July email and [doc](https://docs.google.com/document/d/1_UKje2GuvQhstXWeVsQwHMJ3qc80-bMaTrTMfAP5c4o/edit#):

> Tutorial mode (with overlays explaining how to zoom and maybe even a sample, hand-crafted profile that shows some common stuff)

We may want to do this at a later date when the product is more complete. We likely also want our tutorial to work when integrated with DevTools as well.

Update:
As discussed in the internal meeting with Brian on 20-07-20, a tutorial could be in the form of a short video or something similar to Firefox Profiler's landing with a brief overview of the tool and how to use it.

image

### PR DESCRIPTION
**Pull request by kartikcho**
Resolves #74 by adding key mappings and color legend to the import page. 

This design isn't final and will be redone with a possible new page to help new users get started with using the profiler. (Discussed after the meeting with Brian on 27-07-20)

This will make future iterations easier and is a good step in knowing what instructions to include in the aforementioned 'How to Use' page.

### CODE REVIEW

File: src/ImportPage.css [hunk 1]
```diff
@@ -21,12 +21,35 @@
...
+
+.legendKey {
+  font-size: calc(8px + 1.5vmin);
+}
+
+.legendKey > svg {
+  padding-left: 2rem;
 }
 
+/* Old React spinner animation */
```
Comment (taneliang) [inline src/ImportPage.css]: Could we remove this dead code? Unfortunately it's not easy to locate dead CSS so imo it'll be better to remove them. If we need these styles in the future we can always come back to this PR

File: src/ImportPage.js [hunk 2]
```diff
@@ -76,6 +80,25 @@ export default function ImportPage({onDataImported}: Props) {
                     performance profile
                   </a>{' '}
                   from Chrome Devtools.
+                  <br />
+                  Use <kbd>Ctrl + Scroll</kbd> or <kbd>Shift + Scroll</kbd> to
```
Comment (taneliang) [inline src/ImportPage.js]: This looks like they need to press the Scroll button on their keyboard (which I think may actually exist?). Maybe we should replace this with something like "To zoom, scroll while holding down `Ctrl` or `Shift`."

**Review by taneliang — APPROVED** [review]
Looks good for a first iteration 🚢

File: src/ImportPage.js (hunk 2 above)
Comment (kartikcho) [reply src/ImportPage.js]: Ah, that's a good point.

**Pull request merged**